onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -93,6 +93,15 @@ Module Name:
 
 #define MLAS_UNREFERENCED_PARAMETER(parameter) ((void)(parameter))
 
+//
+// Macro to abort in no exception case, otherwise throw the exception
+//
+#ifdef MLAS_NO_EXCEPTION
+#define MLAS_THROW_EX(ex) abort()
```

Comment:
good adding! print something too :-)

Reply:
I just found there is macro ORT_THROW_EX. Can we just use that? Is there any case that MLAS_NO_EXCEPTION and ORT_NO_EXCEPTIONS are not define or undefined simultaneously.

Reply:
Printed.